### ISSUE TITLE
Kind-of duplicate environments

### ISSUE THREAD

**Issue by thistlillo** [conda/conda#11953]
### Checklist

- [X] I added a descriptive title
- [X] I searched open reports and couldn't find a duplicate

### What happened?

I am speaking about a fresh installation of miniconda `conda 22.9.0` con a Windows 10 Pro PC, with a single environment other than `base`. I have found a [similar issue ](https://github.com/conda/conda/issues/7493), but in their case the duplicate environment was the `base` one. In my case the duplicate environment is the only one I created.

Indeed running `conda env list` in the anaconda command prompt I get:
```
# conda environments:
#
base                   C:\Users\users\miniconda3
ai                  *  C:\Users\user\miniconda3\envs\ai
ai                     c:\Users\user\miniconda3\envs\ai 
```
As you can see, the only difference between the two environments named `ai` is the drive letter, that is uppercase on the first one (second row) and lowercase in the second one (last row). When I run `conda activate ai`, conda activates the environment with the uppercase drive letter.

May be this is related to my using the WSL, but conda is not installed in there.

I am not sure if I can live with it or if it represents a risk of env corruption in the future.

Is there any way to remove one of the two `ai` environments?

### Conda Info

```shell
active environment : base
    active env location : C:\Users\user\miniconda3
            shell level : 1
       user config file : C:\Users\user\.condarc
 populated config files : 
          conda version : 22.9.0
    conda-build version : not installed
         python version : 3.9.12.final.0
       virtual packages : __cuda=11.6=0
                          __win=0=0
                          __archspec=1=x86_64
       base environment : C:\Users\user\miniconda3  (writable)
      conda av data dir : C:\Users\user\miniconda3\etc\conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\user\miniconda3\pkgs
                          C:\Users\user\.conda\pkgs
                          C:\Users\user\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\user\miniconda3\envs
                          C:\Users\user\.conda\envs
                          C:\Users\user\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/22.9.0 requests/2.28.1 CPython/3.9.12 Windows/10 Windows/10.0.19043
          administrator : False
             netrc file : None
           offline mode : False
```

### Conda Config

```shell
-- empty --
```

#### `base` environment:

```shell
# packages in environment at C:\Users\user\miniconda3:
#
# Name                    Version                   Build  Channel
brotlipy                  0.7.0           py39h2bbff1b_1003    defaults
ca-certificates           2022.07.19           haa95532_0    defaults
certifi                   2022.9.24        py39haa95532_0    defaults
cffi                      1.15.1           py39h2bbff1b_0    defaults
charset-normalizer        2.0.4              pyhd3eb1b0_0    defaults
colorama                  0.4.5            py39haa95532_0    defaults
conda                     22.9.0           py39haa95532_0    defaults
conda-package-handling    1.9.0            py39h8cc25b3_0    defaults
console_shortcut          0.1.1                         4    defaults
cryptography              37.0.1           py39h21b164f_0    defaults
idna                      3.3                pyhd3eb1b0_0    defaults
menuinst                  1.4.18           py39h59b6b97_0    defaults
openssl                   1.1.1q               h2bbff1b_0    defaults
pip                       22.2.2           py39haa95532_0    defaults
powershell_shortcut       0.0.1                         3    defaults
pycosat                   0.6.3            py39h2bbff1b_0    defaults
pycparser                 2.21               pyhd3eb1b0_0    defaults
pyopenssl                 22.0.0             pyhd3eb1b0_0    defaults
pysocks                   1.7.1            py39haa95532_0    defaults
python                    3.9.12               h6244533_0    defaults
pywin32                   302              py39h2bbff1b_2    defaults
requests                  2.28.1           py39haa95532_0    defaults
ruamel_yaml               0.15.100         py39h2bbff1b_0    defaults
setuptools                63.4.1           py39haa95532_0    defaults
sqlite                    3.39.3               h2bbff1b_0    defaults
toolz                     0.11.2             pyhd3eb1b0_0    defaults
tqdm                      4.64.1           py39haa95532_0    defaults
tzdata                    2022c                h04d1e81_0    defaults
urllib3                   1.26.11          py39haa95532_0    defaults
vc                        14.2                 h21ff451_1    defaults
vs2015_runtime            14.27.29016          h5e58377_2    defaults
wheel                     0.37.1             pyhd3eb1b0_0    defaults
win_inet_pton             1.1.0            py39haa95532_0    defaults
wincertstore              0.2              py39haa95532_2    defaults
yaml                      0.2.5                he774522_0    defaults
```

#### `ai` environment
```shell
# packages in environment at C:\Users\user\miniconda3\envs\ai:
#
# Name                    Version                   Build  Channel
asttokens                 2.0.8              pyhd8ed1ab_0    conda-forge
attrs                     22.1.0             pyh71513ae_1    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
blas                      1.0                         mkl    defaults
bottleneck                1.3.5            py39h080aedc_0    defaults
brotli                    1.0.9                h2bbff1b_7    defaults
brotli-bin                1.0.9                h2bbff1b_7    defaults
brotlipy                  0.7.0           py39h2bbff1b_1003    defaults
ca-certificates           2022.9.24            h5b45459_0    conda-forge
certifi                   2022.9.24          pyhd8ed1ab_0    conda-forge
cffi                      1.15.1           py39h2bbff1b_0    defaults
charset-normalizer        2.0.4              pyhd3eb1b0_0    defaults
colorama                  0.4.5            py39haa95532_0    defaults
cryptography              37.0.1           py39h21b164f_0    defaults
cuda                      11.8.0                        0    nvidia/label/cuda-11.8.0
cuda-cccl                 11.8.89                       0    nvidia/label/cuda-11.8.0
cuda-command-line-tools   11.8.0                        0    nvidia/label/cuda-11.8.0
cuda-compiler             11.8.0                        0    nvidia/label/cuda-11.8.0
cuda-cudart               11.8.89                       0    nvidia/label/cuda-11.8.0
cuda-cudart-dev           11.8.89                       0    nvidia/label/cuda-11.8.0
cuda-cuobjdump            11.8.86                       0    nvidia/label/cuda-11.8.0
cuda-cupti                11.8.87                       0    nvidia/label/cuda-11.8.0
cuda-cuxxfilt             11.8.86                       0    nvidia/label/cuda-11.8.0
cuda-demo-suite           11.8.86                       0    nvidia/label/cuda-11.8.0
cuda-documentation        11.8.86                       0    nvidia/label/cuda-11.8.0
cuda-libraries            11.8.0                        0    nvidia/label/cuda-11.8.0
cuda-libraries-dev        11.8.0                        0    nvidia/label/cuda-11.8.0
cuda-memcheck             11.8.86                       0    nvidia/label/cuda-11.8.0
cuda-nsight-compute       11.8.0                        0    nvidia/label/cuda-11.8.0
cuda-nvcc                 11.8.89                       0    nvidia/label/cuda-11.8.0
cuda-nvdisasm             11.8.86                       0    nvidia/label/cuda-11.8.0
cuda-nvml-dev             11.8.86                       0    nvidia/label/cuda-11.8.0
cuda-nvprof               11.8.87                       0    nvidia/label/cuda-11.8.0
cuda-nvprune              11.8.86                       0    nvidia/label/cuda-11.8.0
cuda-nvrtc                11.8.89                       0    nvidia/label/cuda-11.8.0
cuda-nvrtc-dev            11.8.89                       0    nvidia/label/cuda-11.8.0
cuda-nvtx                 11.8.86                       0    nvidia/label/cuda-11.8.0
cuda-nvvp                 11.8.87                       0    nvidia/label/cuda-11.8.0
cuda-profiler-api         11.8.86                       0    nvidia/label/cuda-11.8.0
cuda-runtime              11.8.0                        0    nvidia/label/cuda-11.8.0
cuda-sanitizer-api        11.8.86                       0    nvidia/label/cuda-11.8.0
cuda-toolkit              11.8.0                        0    nvidia/label/cuda-11.8.0
cuda-tools                11.8.0                        0    nvidia/label/cuda-11.8.0
cuda-visual-tools         11.8.0                        0    nvidia/label/cuda-11.8.0
cudatoolkit               11.3.1               h59b6b97_2    defaults
cycler                    0.11.0             pyhd3eb1b0_0    defaults
debugpy                   1.6.3            py39h415ef7b_0    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
executing                 1.1.1              pyhd8ed1ab_0    conda-forge
fonttools                 4.25.0             pyhd3eb1b0_0    defaults
freetype                  2.10.4               hd328e21_0    defaults
glib                      2.69.1               h5dc1a3c_1    defaults
gst-plugins-base          1.18.5               h9e645db_0    defaults
gstreamer                 1.18.5               hd78058f_0    defaults
icu                       58.2                 ha925a31_3    defaults
idna                      3.3                pyhd3eb1b0_0    defaults
importlib-metadata        4.11.4           py39hcbf5309_0    conda-forge
importlib_resources       5.10.0             pyhd8ed1ab_0    conda-forge
intel-openmp              2021.4.0          haa95532_3556    defaults
ipykernel                 6.16.0             pyh025b116_0    conda-forge
ipython                   8.5.0              pyh08f2357_1    conda-forge
ipywidgets                8.0.2              pyhd8ed1ab_1    conda-forge
jedi                      0.18.1             pyhd8ed1ab_2    conda-forge
joblib                    1.2.0              pyhd8ed1ab_0    conda-forge
jpeg                      9e                   h2bbff1b_0    defaults
jsonschema                4.16.0             pyhd8ed1ab_0    conda-forge
jupyter_client            7.3.5              pyhd8ed1ab_0    conda-forge
jupyter_core              4.11.1           py39hcbf5309_0    conda-forge
jupyterlab_widgets        3.0.3              pyhd8ed1ab_0    conda-forge
kiwisolver                1.4.2            py39hd77b12b_0    defaults
lerc                      3.0                  hd77b12b_0    defaults
libblas                   3.9.0           1_h8933c1f_netlib    conda-forge
libbrotlicommon           1.0.9                h2bbff1b_7    defaults
libbrotlidec              1.0.9                h2bbff1b_7    defaults
libbrotlienc              1.0.9                h2bbff1b_7    defaults
libcblas                  3.9.0           5_hd5c7e75_netlib    conda-forge
libclang                  12.0.0          default_h627e005_2    defaults
libcublas                 11.11.3.6                     0    nvidia/label/cuda-11.8.0
libcublas-dev             11.11.3.6                     0    nvidia/label/cuda-11.8.0
libcufft                  10.9.0.58                     0    nvidia/label/cuda-11.8.0
libcufft-dev              10.9.0.58                     0    nvidia/label/cuda-11.8.0
libcurand                 10.3.0.86                     0    nvidia/label/cuda-11.8.0
libcurand-dev             10.3.0.86                     0    nvidia/label/cuda-11.8.0
libcusolver               11.4.1.48                     0    nvidia/label/cuda-11.8.0
libcusolver-dev           11.4.1.48                     0    nvidia/label/cuda-11.8.0
libcusparse               11.7.5.86                     0    nvidia/label/cuda-11.8.0
libcusparse-dev           11.7.5.86                     0    nvidia/label/cuda-11.8.0
libdeflate                1.8                  h2bbff1b_5    defaults
libffi                    3.4.2                hd77b12b_4    defaults
libiconv                  1.16                 h2bbff1b_2    defaults
liblapack                 3.9.0           5_hd5c7e75_netlib    conda-forge
libnpp                    11.8.0.86                     0    nvidia/label/cuda-11.8.0
libnpp-dev                11.8.0.86                     0    nvidia/label/cuda-11.8.0
libnvjpeg                 11.9.0.86                     0    nvidia/label/cuda-11.8.0
libnvjpeg-dev             11.9.0.86                     0    nvidia/label/cuda-11.8.0
libogg                    1.3.5                h2bbff1b_1    defaults
libpng                    1.6.37               h2a8f88b_0    defaults
libsodium                 1.0.18               h8d14728_1    conda-forge
libtiff                   4.4.0                h8a3f274_0    defaults
libuv                     1.40.0               he774522_0    defaults
libvorbis                 1.3.7                he774522_0    defaults
libwebp                   1.2.2                h2bbff1b_0    defaults
libxml2                   2.9.14               h0ad7f3c_0    defaults
libxslt                   1.1.35               h2bbff1b_0    defaults
lz4-c                     1.9.3                h2bbff1b_1    defaults
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
matplotlib                3.5.2            py39haa95532_0    defaults
matplotlib-base           3.5.2            py39hd77b12b_0    defaults
matplotlib-inline         0.1.6              pyhd8ed1ab_0    conda-forge
mkl                       2021.4.0           haa95532_640    defaults
mkl-service               2.4.0            py39h2bbff1b_0    defaults
mkl_fft                   1.3.1            py39h277e83a_0    defaults
mkl_random                1.2.2            py39hf11a4ad_0    defaults
msys2-conda-epoch         20160418                      1    conda-forge
munkres                   1.1.4                      py_0    defaults
nbformat                  5.6.1              pyhd8ed1ab_0    conda-forge
nest-asyncio              1.5.6              pyhd8ed1ab_0    conda-forge
nsight-compute            2022.3.0.22                   0    nvidia/label/cuda-11.8.0
numexpr                   2.8.3            py39hb80d3ca_0    defaults
numpy                     1.23.1           py39h7a0a035_0    defaults
numpy-base                1.23.1           py39hca35cd5_0    defaults
openssl                   1.1.1q               h8ffe710_0    conda-forge
packaging                 21.3               pyhd3eb1b0_0    defaults
pandas                    1.4.4            py39hd77b12b_0    defaults
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
pcre                      8.45                 hd77b12b_0    defaults
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.2.0            py39hdc2b20a_1    defaults
pip                       22.2.2           py39haa95532_0    defaults
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_0    conda-forge
ply                       3.11             py39haa95532_0    defaults
prompt-toolkit            3.0.31             pyha770c72_0    conda-forge
psutil                    5.9.0            py39h2bbff1b_0    defaults
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
pycparser                 2.21               pyhd3eb1b0_0    defaults
pygments                  2.13.0             pyhd8ed1ab_0    conda-forge
pyopenssl                 22.0.0             pyhd3eb1b0_0    defaults
pyparsing                 3.0.9            py39haa95532_0    defaults
pyqt                      5.15.7           py39hd77b12b_0    defaults
pyqt5-sip                 12.11.0          py39hd77b12b_0    defaults
pyrsistent                0.18.1           py39hb82d6ee_1    conda-forge
pysocks                   1.7.1            py39haa95532_0    defaults
python                    3.9.13               h6244533_1    defaults
python-dateutil           2.8.2              pyhd3eb1b0_0    defaults
python-fastjsonschema     2.16.2             pyhd8ed1ab_0    conda-forge
python_abi                3.9                      2_cp39    conda-forge
pytorch                   1.12.1          py3.9_cuda11.3_cudnn8_0    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pytz                      2022.1           py39haa95532_0    defaults
pywin32                   303              py39hb82d6ee_0    conda-forge
pyyaml                    6.0              py39h2bbff1b_1    defaults
pyzmq                     23.2.0           py39hd77b12b_0    defaults
qt-main                   5.15.2               he8e5bd7_7    defaults
qt-webengine              5.15.9               hb9a9bb5_4    defaults
qtwebkit                  5.212                h3ad3cdb_4    defaults
requests                  2.28.1           py39haa95532_0    defaults
scikit-learn              1.1.2            py39hfd4428b_0    conda-forge
scipy                     1.8.1            py39h5567194_2    conda-forge
seaborn                   0.11.2             pyhd3eb1b0_0    defaults
setuptools                63.4.1           py39haa95532_0    defaults
sip                       6.6.2            py39hd77b12b_0    defaults
six                       1.16.0             pyhd3eb1b0_1    defaults
sqlite                    3.39.3               h2bbff1b_0    defaults
stack_data                0.5.1              pyhd8ed1ab_0    conda-forge
threadpoolctl             3.1.0              pyh8a188c0_0    conda-forge
tk                        8.6.12               h2bbff1b_0    defaults
toml                      0.10.2             pyhd3eb1b0_0    defaults
torchaudio                0.12.1               py39_cu113    pytorch
torchvision               0.13.1               py39_cu113    pytorch
tornado                   6.2              py39h2bbff1b_0    defaults
tqdm                      4.64.1           py39haa95532_0    defaults
traitlets                 5.4.0              pyhd8ed1ab_0    conda-forge
typing_extensions         4.3.0            py39haa95532_0    defaults
tzdata                    2022c                h04d1e81_0    defaults
urllib3                   1.26.11          py39haa95532_0    defaults
vc                        14.2                 h21ff451_1    defaults
vs2015_runtime            14.27.29016          h5e58377_2    defaults
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
wheel                     0.37.1             pyhd3eb1b0_0    defaults
widgetsnbextension        4.0.3              pyhd8ed1ab_0    conda-forge
win_inet_pton             1.1.0            py39haa95532_0    defaults
wincertstore              0.2              py39haa95532_2    defaults
xz                        5.2.6                h8cc25b3_0    defaults
yaml                      0.2.5                he774522_0    defaults
zeromq                    4.3.4                h0e60522_1    conda-forge
zipp                      3.9.0              pyhd8ed1ab_0    conda-forge
zlib                      1.2.12               h8cc25b3_3    defaults
zstd                      1.5.2                h19a0ad4_0    defaults


### Additional Context

_No response_

**Comment by travishathaway** [comment]
Hi @thistlillo,

This entries are stored in a file that should be at `C:\Users\user\.conda\environments.txt` All you have to do is find the duplicate and remove it.

There is no risk of environment corruption. Sometimes duplicates end up in that file. This is a known issue that has popped up elsewhere.

Thanks for reporting this!

**Comment by travishathaway** [comment]
If you still are having the issue, feel free to re-open!
